tasks/RedHat.yml

```diff
@@ -12,7 +12,7 @@
 - name: Download package locally
   become: no
   connection: local
-  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
+  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} mode=0644 sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
```

Comment:
The problem is when you are required to use a customer control machine where you can't change those permissions. On our case we had umask = 0077